src/components/RulesDialog.vue

```diff
@@ -1,239 +1,216 @@
-            This means your other cards (not the Queen, herself) are protected against Twos (either effect),
-            Nines, and Jacks.
-          </p>
+        <h4>Can I win by playing a King?</h4>
+        <p>
+          Yes. If you meet the required number of points, you win immediately. This means if you play a King
+          when you already have enough points to meet the new limit, you win on that turn.
+        </p>
 
-          <h4>Can I counter a Two with a Two?</h4>
-          <p>
-            Heck yeah! Playing a Two to counter is a One-Off, which can be countered with a two. Stacked
-            counters make for exciting plays.
-          </p>
+        <h4>If I play an Ace or a Six, are my cards destroyed as well?</h4>
+        <p>
+          Yes. Aces and Sixes destroy all Point Cards and all Face Cards plus Glasses Eights, respectively.
+          That includes any that you have out. Try to avoid destroying many of your own cards!
+        </p>
 
-          <h4>Can I win by playing a King?</h4>
-          <p>
-            Yes. If you meet the required number of points, you win immediately. This means if you play a King
-            when you already have enough points to meet the new limit, you win on that turn.
-          </p>
-
-          <h4>If I play an Ace or a Six, are my cards destroyed as well?</h4>
-          <p>
-            Yes. Aces and Sixes destroy all Point Cards and all Face Cards plus Glasses Eights, respectively.
-            That includes any that you have out. Try to avoid destroying many of your own cards!
-          </p>
-
-          <h4>Where can I play Cuttle?</h4>
-          <p>
-            You can play Cuttle anywhere you have a deck of cards and a friend! If you want to play online,
-            there is only one place:
-            <a href="https://www.cuttle.cards" target="_blank">www.cuttle.cards</a>
-          </p>
-        </section>
-      </v-card-text>
-      <v-card-actions class="d-flex justify-end">
-        <v-btn variant="outlined" color="primary" @click="close">Got It!</v-btn>
-      </v-card-actions>
-    </v-card>
-  </v-dialog>
+        <h4>Where can I play Cuttle?</h4>
+        <p>
+          You can play Cuttle anywhere you have a deck of cards and a friend! If you want to play online,
+          there is only one place:
+          <a href="https://www.cuttle.cards" target="_blank">www.cuttle.cards</a>
+        </p>
+      </section>
+    </template>
+    <template #actions>
+      <v-btn variant="outlined" color="surface-1" @click="close">Got It!</v-btn>
+    </template>
+  </base-dialog>
 </template>
 
 <script>
+import BaseDialog from '@/components/Global/BaseDialog.vue';
+
 export default {
   name: 'RulesDialog',
   emits: ['close'],
+  components: {
+    BaseDialog,
+  },
   props: {
     show: {
       type: Boolean,
       default: false,
     },
-    hideActivator: {
-      type: Boolean,
-      default: false,
-    },
   },
```

Comment:
Is the `show` prop actually used? I don't see it in the base-dialog instance so I'm wondering if this actually opens atm?